store/sqlstore/post_store.go

```diff
@@ -143,6 +144,18 @@ func (s *SqlPostStore) SaveMultiple(posts []*model.Post) ([]*model.Post, int, er
 		}
 
 		if post.RootId == "" {
+			currentChannelCount, ok2 := channelNewRootPosts[post.ChannelId]
+			if !ok2 {
```

Comment:
Purely stylistic suggestion (feel free to ignore, but I've been writing some more Go myself recently and started to use this pattern and it avoids the mess of creating a second `ok` variable and the `currentChannelCount` will also be available in the `else` clause):

```
if currentChannelCount, ok := channelNewRootPosts[post.ChannelId]; ok {
   ...
} else {
   ...
}
```